.ci/mellanox/azure-pipelines.yml

```diff
@@ -0,0 +1,76 @@
+# Per-commit
+trigger:
+  - master
+# All PRs
+pr:
+  branches:
+    include:
+    - '*'
+
+pool:
+  name: Default
+  demands: CUSTOM_CAPABILITY -equals ompi
+
+variables:
+  ompi_ci_os_name: centos
+  ompi_ci_os_version: 7
+  ompi_ci_ofed: 4.7-1.0.0.1
+  ompi_ci_username: swx-jenkins
+  ompi_ci_uid: 6213
+  ompi_ci_gid: 11429
+  # Enable debug information, supported values: true, false
+  debug: true
+
+jobs:
+- job: mellanox_ompi_ci
+  displayName: Mellanox OpenMPI CI
+  timeoutInMinutes: 240
+  pool:
+    name: Default
+    demands: CUSTOM_CAPABILITY -equals ompi
+  steps:
+  - task: DeleteFiles@1
+    displayName: Cleanup workspace folder
+    inputs:
+      sourceFolder: $(Pipeline.Workspace)
+      contents: |
+        **/*
+  - checkout: self
+    submodules: true
+  - bash: |
```

Comment:
IMO we should build the Docker image once and put it to dockerhub or any other public registry. Building the image every PR increases CI time and makes it less reliable. WDYT?

Reply:
This's a temporary solution while there aren't any docker registries available. At the current moment Mellanox Harbor docker registry is available - the pipeline is to be updated. Also as we discussed with @amaslenn we should consider Mellanox Dockerhub usage to make building docker images available for external users (e.g. to reproduce building environment as much as possible).